src/meta/test/meta_app_operation_test.cpp

```diff
@@ -22,6 +22,7 @@
 #include "meta_service_test_app.h"
 #include "meta_test_base.h"
 #include "meta/meta_split_service.h"
+#include "misc/misc.h"
```

Comment:
Why would you add this header file?

Reply:
`misc/misc.h` is included to use `random32` to choose a random partition in L431 ~